### ISSUE TITLE
Table view inside drawer 

### ISSUE THREAD

**Issue by thewriterl** [52inc/Pulley#295]
Hello, 

I'm trying to put a table view inside the drawer programmatically but I can't scroll it, every time I scroll, the drawer gets scrolled. Don't have much code besides declarations 

`let mainContentVC = MainViewController()
        
        let drawerContentVC = DrawerViewController()
                
        let pulleyController = PulleyViewController(contentViewController: mainContentVC, drawerViewController: drawerContentVC)
        `

and this on DrawerViewController class

` let tableView: UITableView = {
        let tb = UITableView()
        return tb
    }()
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        view.backgroundColor = .blue
        view.addSubview(searchInput)
        view.addSubview(tableView)
`

**Comment by brendan09** [comment]
Hi– Checkout the sample project. It does exactly this.

**Comment by thewriterl** [comment]
actually it doesn't, it instantiates a storyboard with an identifier.

**Comment by brendan09** [comment]
Yes, which has a table view in it. You can see any code or settings relating to the table view and how it’s managed in the drawer view controller in the sample project.

**Comment by thewriterl** [comment]
I don't use storyboards, how can I use Pulley?

**Comment by brendan09** [comment]
Pulley doesn’t require storyboards. The sample simply loads a VC from a storyboard (you can see it manually initialize with view controllers in AppDelegate, although it’s commented out). 

Where the view controller comes from is irrelevant, storyboard or not storyboard, it works the same. 

You can look at the drawer view controller in the sample project to see what’s being done with the table view and in interface builder to see any non-code settings and the view hierarchy.
